app/gui/src/project-view/components/CircularMenu.vue

```diff
@@ -39,64 +49,75 @@ function readableBinding(binding: keyof (typeof graphBindings)['bindings']) {
   <div
     class="CircularMenu"
     :class="{
-      partial: !props.isFullMenuVisible,
       menu: !showColorPicker,
+      openedDropdown: isDropdownOpened,
     }"
   >
     <template v-if="!showColorPicker">
-      <template v-if="isFullMenuVisible">
-        <SvgButton
-          v-if="documentationUrl"
-          name="help"
-          class="slot1"
-          :title="`Open Documentation (${readableBinding('openDocumentation')})`"
-          @click.stop="openDocs(documentationUrl)"
-        />
-        <SvgButton
-          name="comment"
-          class="slot2"
-          title="Comment"
-          @click.stop="emit('startEditingComment')"
-        />
-        <SvgButton
-          name="paint_palette"
-          class="slot3"
-          title="Color"
-          @click.stop="showColorPicker = true"
-        />
-        <SvgButton
-          name="trash2"
-          class="slot4"
-          :title="`Delete (${readableBinding('deleteSelected')})`"
-          data-testid="removeNode"
-          :disabled="!isRemovable"
-          @click.stop="emit('delete')"
-        />
-      </template>
-      <div v-else class="More" @pointerdown.stop="emit('openFullMenu')"></div>
-      <ToggleIcon
-        icon="eye"
-        class="slot5"
+      <SvgButton
+        name="eye"
+        class="slot1"
```

Comment:
Now I think we could name it from directions like "slot1 -> slot_S", "slot2 -> slot_SW" etc.

Reply:
Done